examples/09_cricket_umpire/README.md

```
@@ -0,0 +1,179 @@
+
+---
+
+## 🎥 Demo Video
+
+[▶ Watch Demo](https://youtu.be/j3rzsTp0sW0)
+
+## 🌐 Deployment
+
+| | Link |
+|---|---|
+| Frontend | [Link](https://ai-drs-vision-agents-sdk-jd.vercel.app/) |
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)